parm/archive/gfs_arcdir.yaml.j2

```diff
@@ -108,6 +108,6 @@ copy_opt:
        "{{ ARCDIR }}/abias_air.{{ RUN }}.{{ cycle_YMDH }}"]
     - ["{{ COMIN_ATMOS_ANALYSIS }}/{{ head }}abias_int.txt",
        "{{ ARCDIR }}/abias_int.{{ RUN }}.{{ cycle_YMDH }}"]
-    - ["{{ COMIN_ATMOS_ANALYSIS }}/{{ head }}analysis.dtf.a006.nc",
+    - ["{{ COMIN_ATMOS_ANALYSIS }}/{{ head }}increment.dtf.i006.nc",
        "{{ ARCDIR }}/dtfanl.{{ RUN }}.{{ cycle_YMDH }}.nc"]
```

Comment:
```suggestion
       "{{ ARCDIR }}/dtfinc.{{ RUN }}.{{ cycle_YMDH }}.nc"]
```

Reply:
Since `dtfanl` files are actually increments instead of analysis files I think the long term solution should be that everything gets renamed to `dtfinc`. Please correct me if I'm wrong on that. I just assume a new user will come down the road later and see the `dtfanl.nc` output and assume they are analysis files. But making that change is probably a bigger scope than what this PR intended as this is primarily a fix for GCAFS. If you want to try and tackle it here you can, otherwise I suggest modifying the description to highlight you renamed internal NSST files and we can have another issue opened to look into renaming `dtfanl`.

Reply:
Ok I'll open an issue that is more general to rename dtfanl to dtfinc everywhere

Reply:
See https://github.com/NOAA-EMC/global-workflow/issues/4395